test/gtest/common/xpmem_test.h

```diff
@@ -91,10 +72,36 @@ class xpmem_test {
   static int pattern_next(int seed);
 
   static std::vector<int> randomized_sequence(int count);
-  std::vector<std::pair<uint8_t*, size_t>> mmap_holes(size_t hole_size,
-                                                      size_t size, int count,
-                                                      bool fault_pages = true);
-  void munmap_holes(std::vector<std::pair<uint8_t*, size_t>> &areas);
-  size_t mmap_holes_sum(size_t size, int count);
+
+ public:
+  static void *mmap(size_t size, int prot = PROT_READ | PROT_WRITE,
+                    int flags = MAP_PRIVATE | MAP_ANONYMOUS) {
```

Comment:
maybe define costs
DEFAULT_MMAP_PROT
DEFAULT_MMAP_FLAGS

Reply:
added static const in class